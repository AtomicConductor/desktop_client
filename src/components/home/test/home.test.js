import React from "react";

import { shallow } from "enzyme";
import Home from "../index";

it("renders without crashing", () => {
  shallow(<Home />);
});

describe("snapshots", () => {
  it("match snapshot when status is success", () => {
    expect(shallow(<Home />)).toMatchSnapshot();
  });
});
