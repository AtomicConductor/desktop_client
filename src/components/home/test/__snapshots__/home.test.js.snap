// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots match snapshot when status is success 1`] = `
<Styled(MuiBox)
  className="makeStyles-container-1"
>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-headerText-2"
    variant="h4"
  >
    Get Started!
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Typography))
    className="makeStyles-subheader-3"
    color="textSecondary"
    variant="h6"
  >
    Conductor tools are installed and you are ready to go. Use the links below to get specific info for your platform or to contact us for help.
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Grid))
    container={true}
    spacing={8}
  >
    <IndexCard
      body={
        <React.Fragment>
          Learn how to submit your renders from within 
          <strong>
            Maya
          </strong>
          , 
          <strong>
            Nuke
          </strong>
           and 
          <strong>
            Clarisse
          </strong>
          , or get help crafting your submissions through simple Python APIs.
        </React.Fragment>
      }
      icon={<Memo(LibraryBooksRoundedIcon) />}
      onClick={[Function]}
      title="Documentation"
    />
    <IndexCard
      body="Have questions about the service or need some guidance? Submit a support ticket and you'll get a response from one of our developers within the hour."
      icon={<Memo(ContactSupportRoundedIcon) />}
      onClick={[Function]}
      title="Support center"
    />
    <IndexCard
      body="Head over to our web application at conductortech.com to check the status of your running jobs and more."
      icon={<Memo(WebRoundedIcon) />}
      onClick={[Function]}
      title="Web Dashboard"
    />
    <IndexCard
      body="Thanks for trying out the Conductor desktop agent. If you find an issue or have a suggestion, please click here to let us know."
      icon={<Memo(FeedbackRoundedIcon) />}
      onClick={[Function]}
      title="Beta Feedback"
    />
    <_default
      feebackFormOpen={false}
      onCloseFeedbackForm={[Function]}
    />
  </WithStyles(ForwardRef(Grid))>
</Styled(MuiBox)>
`;
